src/Uno.Extensions.Reactive.UI/Presentation/Bindings/BindableViewModelBase.cs

```diff
@@ -54,37 +63,52 @@ protected BindablePropertyInfo<TProperty> Property<TProperty>(string propertyNam
 
 		return info;
 
-		void ViewModelToView(Action<TProperty?> updated)
-			=> DispatcherHelper.TryEnqueue(dispatcher, async () =>
-			 {
-				 try
-				 {
-					 updated(defaultValue);
-
-					// Note: No needs to use .WithCancellation() here as we are enumerating the stateImp which is going to be disposed anyway.
-					await foreach (var msg in stateImpl.GetSource().ConfigureAwait(true))
-					 {
-						 if (msg.Current.Get(BindingSource) != this)
-						 {
-							 updated(msg.Current.Data.SomeOrDefault());
-						 }
-					 }
-				 }
-				 catch (Exception error)
-				 {
-					 this.Log().Error(
-						 $"Synchronization from ViewModel to View of '{propertyName}' failed."
-						 + "(This is a final error, changes made in the VM are no longer propagated to the View.)",
-						 error);
-				 }
-			 });
+		async void ViewModelToView(Action<TProperty?> updated)
```

Comment:
Should it be cancellable?

Reply:
On dispose the `stateImpl` will be disposed, which will end the `source` async enumerable. We could however also cancel the async dispatcher resolution ... pushing a new commit for that!